sys/kern/kern_exit.c

```diff
@@ -854,16 +853,19 @@ sys_wait6(struct thread *td, struct wait6_args *uap)
 	error = user_wait6(td, uap->idtype, uap->id, uap->status, uap->options,
 	    uap->wrusage, sip);
 	if (uap->info != NULL && error == 0) {
-		siginfo_to_siginfo_native(&si, &si_n);
-		error = copyout(&si_n, uap->info, sizeof(si_n));
+		/*
+		 * XXX: Should we document that we are explicitly
```

Comment:
I think we strip here because we are probably returning siginfo to another process.  I do think it might be worth explicitly documenting the cases when we choose to strip tags by using copyout() instead of copyoutcap().  It's somewhat unfortunate in this case that we have no good way to pass the tags in a sidecar in siginfo_t.

Reply:
Actually, wait6() should only return `SIGCHLD` signal information which should never have any capabilities to return, so I've updated the comment to reflect that.